.github/workflows/mingw.yml

```diff
@@ -50,8 +50,8 @@ jobs:
         shell: msys2 {0}
         run: |
           set -ex
-          pkgver='3.9.5'
-          _pybasever='3.9'
+
+          CFLAGS="-O2"
```

Comment:
Maybe if its possible it would be better to have it inside the build system rather than a CFLAGS env var. But its non-blocking.

Reply:
Not sure..  Increasing the stack size would be safer, but I don't know if there is any downside.

Reply:
I've created https://github.com/msys2-contrib/cpython-mingw/issues/13